src/actions/utxo.js

```diff
@@ -2,48 +2,84 @@
 
 import walletManager from '../crypto/wallet'
 
+import type {RawUtxo} from '../types/HistoryTransaction'
+
 import type {Dispatch} from 'redux'
 
 import type {State} from '../state'
 
+declare type ExtractReturnType = <R>((...arg: any) => R) => R;
```

Comment:
These two types shouldn't go here -- it's just convenient to place them here for this example.

They should go in a Flow utils folder and then added to the `[libs]` section of `.flowconfig` like we have in the Yoroi extension